lib/brexit_checker/actions.yaml

```diff
@@ -337,7 +337,7 @@ actions:
   guidance_url: https://www.gov.uk/guidance/prepare-to-use-the-ukca-mark-after-brexit
   criteria:
   - any_of:
-    - sell-goods-provide-services-in-uk
+    - owns-operates-business-organisation
```

Comment:
Need to check implications of this.

Reply:
Hmm...this is suspicious. If we have this criteria, we shouldn't need the export one, since it's not possible to have the latter without the former 🤔.

Reply:
That's a great point, I guess this is implicit by virtue of question criteria.
Let me check in again with Mark, not sure if this choice is about the results page.

Reply:
@benthorner Ah ha ok, caught up with mark so the criteria goes something like this.
Given that you _do_ own or operate a business in the UK
And that you _do not_ export to the eu

You still need to check CE guidance as there could be a new UK CA stamp that could be relevant.

Reply:
Guidance is "In some cases you will need to apply the UKCA marking to goods being sold in the UK immediately after Brexit"

Reply:
I'm still confused, but maybe I wasn't clear before. I'll try explain slightly differently: 'owns-operates-...' is a superset of 'export-to-eu', since it's not possible to 'export-to-eu' unless you 'owns-operates-...'. Given this action uses an `any_of` predicate, it's not necessary to have 'export-to-eu'. 

I notice that the English version in your comment above is different to the actual criteria expression, so maybe someone needs to fix that?

Reply:
Could I gather you, mark and I together in person on this one?
I think we're losing quite a bit in translation?

Reply:
Have amended the copy above so it's an exact quote of the guidance and not my sloppy typing of what was being read out loud

Reply:
Ok resolved. We do not need the exporting to EU. 
And as this enlarges the scope of notifications we won't need to change any subscriptions in SQL